compiler/damlc/daml-stdlib-src/DA/Internal/Exception.daml

```diff
@@ -137,4 +151,8 @@ instance ActionCatch Update where
 
 --------------------------------------------------------------
 
+-- | HIDE
+handleBuiltinException : Text -> (Text -> e) -> (AnyException -> Optional e) -> (AnyException -> Optional e)
+handleBuiltinException _ _ f = f -- magic @"handleBuiltinException"
```

Comment:
what’s up with the comment?

Reply:
It's a placeholder for adding some rules in LFConversion for dealing with builtin exceptions (rather than handling them in daml).

Reply:
maybe add a comment?